extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -486,4 +487,14 @@ private int getAlertDuration(Command command) {
 
         return delay;
     }
+
+    @Override
+    public void bridgeStatusChanged(ThingStatusInfo bridgeStatusInfo) {
+        super.bridgeStatusChanged(bridgeStatusInfo);
+        if (ThingStatus.ONLINE.equals(bridgeStatusInfo.getStatus())
+                && ThingStatus.OFFLINE.equals(getThing().getStatusInfo().getStatus())
+                && ThingStatusDetail.BRIDGE_OFFLINE.equals(getThing().getStatusInfo().getStatusDetail())) {
+            updateStatus(ThingStatus.ONLINE, ThingStatusDetail.NONE);
```

Comment:
As mentioned in https://github.com/eclipse/smarthome/issues/4074#issuecomment-323281682, I would claim that setting the status ONLINE is many cases a bug. In the example of the hue binding, the handler determines the correct status in `onLightStateChanged`. So either we do not set a state here at all and wait for the next update (triggered through the regular polling) or we set it to UNKNOWN for the time being.

Same should be checked for the other bindings.

Reply:
I checked that and changed the PR.